dopamine/replay_memory/circular_replay_buffer.py

```diff
@@ -192,6 +201,23 @@ def _create_storage(self):
       array_shape = [self._replay_capacity] + list(storage_element.shape)
       self._store[storage_element.name] = np.empty(
           array_shape, dtype=storage_element.type)
+    self._trajectories = []
+    self._trajectory_lengths = []
```

Comment:
Are lengths used in a later commit?  I don't see them used in this one.

FWIW, it seems marginally less safe to use manual lengths rather than `len(self._trajectory[i])`.

Reply:
ignoring this as lengths are removed in the next commit. This is because I went from one implementation to another and forgot to remove.